app/views/admin/users/_current_roles.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="crayons-card p-6 grid gap-2">
+  <h2>Current Roles</h2>
+  <p>(View full history in notes below)</p>
+  <div class="crayons-card__body">
+    <ul>
+      <% @user.roles.each do |role| %>
+        <% if role.name == "banned" %>
+          <li>Suspended</li>
+        <% elsif role.name == "super_admin" || current_user.id == @user.id %>
+          <li><%= role.name.titlecase %></li>
+        <% else %>
+          <li><%= role.name.titlecase %> <em><%= role.resource_name.to_s %></em> &nbsp;<%= link_to "REMOVE", url_for(action: :destroy, user_id: @user.id, role: role.name.to_sym), method: :delete, data: { confirm: "Are you sure?" },
+                                                                                                                                                                                   class: "crayons-btn crayons-btn--danger crayons-btn--s lh-base fs-xs" %></li>
```

Comment:
**nitpick (non-blocking):** Maybe we can look into formatting this a bit differently, the content of this line starts _past_ what is supposed to be our maximum line length 😅